core/templates/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -26,6 +26,8 @@ interface UiConfig {
   (): UiConfig;
```

Comment:
Can you also confirm that the saved rte HTML with rtl orientation is also renderers in the same order in the editor/viewer page?

Reply:
I tested this, it does.